.github/workflows/build.yml

```diff
@@ -222,3 +222,30 @@ jobs:
       with:
         name: reactos-msvc-amd64-${{github.sha}}
         path: build/livecd.iso
+
+  build-msbuild-i386:
+    name: MSBuild (i386)
+    runs-on: windows-latest
+    steps:
+    - name: Install Flex and Bison
+      run: |
+        curl -O https://svn.reactos.org/storage/vperevertkin/flexbison.7z
+        7z x flexbison.7z -O${{github.workspace}}\bin
+        echo "${{github.workspace}}\bin" >> $GITHUB_PATH
+        echo "BISON_PKGDATADIR=${{github.workspace}}\bin\share\bison" >> $GITHUB_ENV
+        echo "M4=${{github.workspace}}\bin\m4.exe" >> $GITHUB_ENV
```

Comment:
wouldnt it be easier to add rosbe?

Reply:
Something like what I use to launch a shell:
```
call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvarsall.bat" x86
set Platform=
set PATH=C:\RosBE\Bin;%PATH%
configure.cmd VSSolution -DENABLE_ROSTESTS=1 -DENABLE_ROSAPPS=1
```

Reply:
I don't download RosBE on Windows builders, because the only needed things from there are Flex and Bison, everything else is installed already